sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultTransportEncryptionSpec.java

```diff
@@ -0,0 +1,84 @@
+package com.mesosphere.sdk.specification;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.mesosphere.sdk.specification.validation.ValidationUtils;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import javax.validation.Valid;
+
+/**
+ * Default implementation of {@link TransportEncryptionSpec}.
+ */
+public class DefaultTransportEncryptionSpec implements TransportEncryptionSpec {
+
+    @Valid
+    private String name;
```

Comment:
(The `@valid` simply tells the validation to ensure that `name` is validated without actually specifying what the validators are)